src/traces/sankey/attributes.js

```diff
@@ -9,13 +9,27 @@
 'use strict';
 
 var fontAttrs = require('../../plots/font_attributes');
+var plotAttrs = require('../../plots/attributes');
 var colorAttrs = require('../../components/color/attributes');
 var fxAttrs = require('../../components/fx/attributes');
 var domainAttrs = require('../../plots/domain').attributes;
 
+var extendFlat = require('../../lib/extend').extendFlat;
 var overrideAll = require('../../plot_api/edit_types').overrideAll;
 
-var attrs = module.exports = overrideAll({
+module.exports = overrideAll({
+    hoverinfo: extendFlat({}, plotAttrs.hoverinfo, {
+        flags: [],
```

Comment:
Nicely done, `plots.hoverinfo` already has the correct `extras` field

https://github.com/plotly/plotly.js/blob/d5ebb50a727821f6705d72182d7822d0097e8a41/src/plots/attributes.js#L121